### PR TITLE
Rework payments descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks payments descriptions, and related files to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 1]
```diff
@@ -116,53 +123,63 @@ properties:
         - sca
       x-enumDescriptions:
         authorize: |-
-          Creates an `authorize` transaction in the amount/currency of the request.
-          If the gateway account is configured for strong customer authentication, it is used.
+          Creates an `authorize` transaction in the amount or currency of the request.
+          This is used when a gateway account is configured for Strong Customer Authentication (SCA)
```
Comment (adamaltman) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:127]:
```suggestion
          This is used when a gateway account is configured for Strong Customer Authentication (SCA).
```

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 2]
```diff
@@ -53,29 +56,33 @@ properties:
...
           description: Short description of the target documentation.
           type: string
   publishedPricing:
-    description: If they have published pricing, a description of it here.
+    description: Pricing description for the payment gateway, if pricing is published.
     type: string
   setupInstructions:
     description: |-
-      Some special instructions for merchant to setup account for this gateway.
+      Special gateway account set up instructions for merchant.
```
Comment (noon-dawg) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:72]:
```suggestion
      Special gateway account set up instructions for merchants.
```

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 3]
```diff
@@ -100,12 +107,12 @@ properties:
         - 3DS2
         - checkCredentials
   supported3dsServers:
-    description: Array of the 3DS-servers supported.
+    description: Array of support 3DS-servers.
```
Comment (noon-dawg) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:110]:
```suggestion
    description: Array of supported 3DS-servers.
```

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 4]
```diff
@@ -116,53 +123,63 @@ properties:
...
+    description: |-
+      Specifies whether offsite interaction is required.
+      For example, 3DS, PayPal login, or completing something in a store.
     type: boolean
     default: false
   ipn:
     type: object
-    description: Describes the IPN that a gateway supports. No value means IPN is not supported.
+    description: |-
+      Describes the IPN that a payment gateway supports.
```
Comment (noon-dawg) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:158]:
```suggestion
      Describes which Instant Payment Notification (IPN) a payment gateway supports.
```

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 5]
```diff
@@ -116,53 +123,63 @@ properties:
...
+      If this value is null, IPN is not supported.
     required:
       - type
     properties:
       type:
-        description: Type of supported IPN. No value means IPN is not supported.
+        description: |-
+          Type of supported IPN.
```
Comment (noon-dawg) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:165]:
```suggestion
          Type of supported Instant Payment Notification (IPN) .
```

File: openapi/components/schemas/PaymentCards/PaymentCardUpdatePlain.yaml [hunk 6]
```diff
@@ -1,20 +1,25 @@
 type: object
 properties:
   cvv:
-    description: Card cvv (card verification value).
+    description: Card Verification V (CVV) of the payment card.
```
Comment (noon-dawg) [inline openapi/components/schemas/PaymentCards/PaymentCardUpdatePlain.yaml:4]:
```suggestion
    description: Card Verification Value (CVV) of the payment card.
```

File: openapi/paths/storefront/payment-instruments@{id}@setup.yaml [hunk 7]
```diff
@@ -32,22 +32,22 @@ post:
...
     - CustomerJWT: []
   description: |-
-    Create the payment instrument's setup transaction.
+    Creates a set up payment instrument transaction.
 
-    This effectively makes the payment instrument available for further payments.
-    The response should be treated as a regular transaction;
-    for example, the approval links should be followed until the transaction is completed.
+    This operation makes the payment instrument available for further payments.
+    The response must be treated as a regular transaction.
```
Comment (noon-dawg) [inline openapi/paths/storefront/payment-instruments@{id}@setup.yaml:44]:
```suggestion
    Treat the response as a regular transaction.
```

File: openapi/paths/storefront/payment-instruments@{id}@setup.yaml [hunk 8]
```diff
@@ -32,22 +32,22 @@ post:
...
+    For example, the approval links must be followed until the transaction is completed.
```
Comment (noon-dawg) [inline openapi/paths/storefront/payment-instruments@{id}@setup.yaml:45]:
```suggestion
    For example, approval links must be followed until the transaction is completed.
```

File: openapi/openapi.yaml [hunk 9]
```diff
@@ -300,26 +300,35 @@ tags:
...
       A subscription order generates an invoice for each service period.
       For more information, see [Orders](https://www.rebilly.com/docs/dev-docs/concepts/#orders).
   - name: Usage
-    description: Reports the product usage of a subscription item for metered billing purposes.
+    description: |-
+      Use these operations to manage the product usage of a subscription item for metered billing purposes.
+
+      Use metered billing when product quantity is unknown to the customer at the moment of creating a subscription.
+      Metered billing is based on reported usage records.
+      Every reported usage updates the quantity of an upcoming invoice item for a specified subscription and a plan. To create a metered billing plan, [Create a plan](../../PostPlan).
```
Comment (noon-dawg) [inline openapi/openapi.yaml:318]:
```suggestion
      Every reported usage updates the quantity of an upcoming invoice item for a specified subscription and a plan. 
      To create a metered billing plan, [Create a plan](../../PostPlan).
```

File: openapi/components/schemas/Common/CommonTransaction.yaml [hunk 10]
```diff
@@ -87,7 +88,10 @@ properties:
     allOf:
       - $ref: ../CurrencyCode.yaml
   requestAmount:
-    description: Amount of the payment request. If an adjustment occurs, the purchased amount may differ from the billing amount.
+    description: |-
+      Amount of the payment request.
+      If an adjustment occurs,
+      the purchased amount may differ from the billing amount.
```
Comment (adamaltman) [inline openapi/components/schemas/Common/CommonTransaction.yaml:94]:
```suggestion
      the purchase amount may differ from the billing amount.
```
I think this means literally the `purchaseAmount` field above.

File: openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml [hunk 11]
```diff
@@ -22,26 +22,32 @@ properties:
...
+          Bank Identification Number (BIN) of the payment card.
+          This is usually the first 6 digits of the payment card number.
         type: string
         format: bin
         readOnly: true
       last4:
-        description: Payment Card PAN last 4 digits.
+        description: |-
+          Permanent Account Number (PAN) of the payment card.
+          This is usually the last 4 digits of the payment card number.
```
Comment (adamaltman) [inline openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:50]: It is. Why usually?

File: openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml [hunk 12]
```diff
@@ -22,26 +22,32 @@ properties:
...
+          Bank Identification Number (BIN) of the payment card.
+          This value is the first 6 digits of the payment card number.
         type: string
         format: bin
         readOnly: true
       last4:
-        description: Payment Card PAN last 4 digits.
+        description: |-
+          Permanent Account Number (PAN) of the payment card.
+          This value is the last 4 digits of the payment card number.
```
[the remark this answers is not in the document]
Comment (noon-dawg) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:50]: Nice! This is much more concise

File: openapi/paths/storefront/payment-instruments@{id}@setup.yaml [hunk 13]
```diff
@@ -32,22 +32,22 @@ post:
     - Storefront
   tags:
     - Payment instruments
-  summary: Create payment instrument setup transaction
+  summary: Create a set up payment instrument transaction
```
Comment (adamaltman) [inline openapi/paths/storefront/payment-instruments@{id}@setup.yaml:35]: It's actually a transaction type we call `setup` one word.

File: openapi/paths/storefront/payment-instruments@{id}@setup.yaml [hunk 14]
```diff
@@ -5,18 +5,18 @@ get:
     - Storefront
   tags:
     - Payment instruments
-  summary: Retrieve payment instrument setup transaction
+  summary: Retrieve a payment instrument set up transaction
```
Comment (adamaltman) [inline openapi/paths/storefront/payment-instruments@{id}@setup.yaml:8]:
```suggestion
  summary: Retrieve a payment instrument setup transaction
```
It's a transaction type like authorize, capture, named: `setup`.

Comment (noon-dawg) [reply openapi/paths/storefront/payment-instruments@{id}@setup.yaml]: Updated in [005ee09](https://github.com/Rebilly/api-definitions/pull/1166/commits/005ee0916f6d0b5b02926de0c48739f2d1ce17b7)

**Comment by noon-dawg** [comment]
@Rebilly/php please review

File: openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml [hunk 15]
```diff
@@ -53,29 +56,33 @@ properties:
...
-    description: Array of the payment methods supported (for example, `payment-card`, `bitcoin`).
+    description: Array of supported payment methods. For example, `payment-card` and `bitcoin`.
     type: array
     items:
       $ref: ./PaymentMethodApiName.yaml
     minLength: 1
   cardBrands:
-    description: Array of the card-brands supported, if `payment-card` is a value of paymentMethods.
+    description: |-
+      Array of support card-brands.
```
Comment (Boydbueno) [inline openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml:84]:
```suggestion
      Array of supported card-brands.
```

File: openapi/paths/storefront/payment-instruments@{id}@setup.yaml [hunk 16]
```diff
@@ -32,17 +32,17 @@ post:
     - Storefront
   tags:
     - Payment instruments
-  summary: Create payment instrument setup transaction
+  summary: Create a set up payment instrument transaction
```
Comment (Boydbueno) [inline openapi/paths/storefront/payment-instruments@{id}@setup.yaml:35]: As Adam mentioned, `setup` is the transaction type.
```suggestion
  summary: Create a setup payment instrument transaction
```

**Review by Boydbueno — DISMISSED** [review]

File: openapi/components/requestBodies/Webhooks/PaymentCardWebhook.yaml [hunk 17]
```diff
@@ -3,7 +3,7 @@ content:
     schema:
       properties:
         paymentCardId:
-          description: Payment card ID.
+          description: ID of the payment.
```
Comment (ashkarpetin) [inline openapi/components/requestBodies/Webhooks/PaymentCardWebhook.yaml:6]:
```suggestion
          description: ID of the payment instrument.
```

File: openapi/components/schemas/PaymentInstruments/PaymentInstrumentCreateToken.yaml [hunk 18]
```diff
@@ -7,7 +7,7 @@ properties:
   customerId:
     $ref: ../CustomerId.yaml
   token:
-    description: Payment token ID.
+    description: Token ID of the payment.
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentInstruments/PaymentInstrumentCreateToken.yaml:10]:
```suggestion
    description: ID of the payment token.
```

File: openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml [hunk 19]
```diff
@@ -76,11 +80,11 @@ properties:
   updatedTime:
     $ref: ../UpdatedTime.yaml
   usageTime:
-    description: Token usage time.
+    description: Date and time when the token is used.
     allOf:
       - $ref: ../ServerTimestamp.yaml
   expirationTime:
-    description: Token expiration time.
+    description: Date and time when the token expires.
```
Comment (ashkarpetin) [inline openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:87]:
```suggestion
    description: Date and time when the token is expired.
```

?

Comment (noon-dawg) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:87]: I think this is okay. Where possible, we should use current tense and avoid passives

Comment (ashkarpetin) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:87]: `expirationTime` doesn't show when the token will expire, it is updated only when token is expired

Comment (noon-dawg) [reply openapi/components/schemas/PaymentTokens/PaymentCardToken.yaml:87]: Ah! It should be `expired` then. Thanks for pointing this out 🙏

**Review by swilczynski — APPROVED** [review]

**Review by Boydbueno — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**